public/styles/dark.css

```diff
@@ -23,7 +23,7 @@
     --color-label-tags: var(--TBD);
     --color-label-shortcut: var(--TBD); */
 
-    --text-color-primary: var(--harbor-gray);
+    --text-color-primary: var(--pearl-river);
```

Comment:
This is dangerous - please verify this change very good !

Reply:
Thanks for the comment and the warning! 👍🏼 
Checked it on many places now, a big lighter color. An almost invisible change if you are not told it's been changed